tests/test_download.py

```diff
@@ -310,3 +310,47 @@ def other_service(_):
                 response.json["error"]
                 == f"Service with index={service.id} is not an access service."
             )
+
+
+def test_download_arweave(client, publisher_wallet, consumer_wallet, web3):
+    unencrypted_files_list = [
+        {
+            "type": "arweave",
+            "transactionId": "cZ6j5PmPVXCq5Az6YGcGqzffYjx2JnsnlSajaHNr20w",
```

Comment:
I'm not very familiar with arweave, I assume this is a known transaction that you have created. Is it permanent? Can it disappear someday? Can you add some details on that?

Reply:
Sure, I'll add a comment describing the transaction.

Yes, I created this transaction, and assuming the Arweave network operates as intended, it is permanent and will never disappear.  This particular transaction happens to be a blog post that I published using https://mirror.xyz/

Here's the Arweave gateway URL:  https://ogpkhzhzr5kxbkxebt5gazygvm356yr4oythwj4ve2rwq43l3nga.arweave.net/cZ6j5PmPVXCq5Az6YGcGqzffYjx2JnsnlSajaHNr20w

Reply:
I removed all the mentions of the blog post transaction id and instead added `tests/helpers/constants.py` which contains:

```python
"""branin.arff dataset, permanently stored in Arweave"""
ARWEAVE_TRANSACTION_ID = "a4qJoQZa1poIv5guEzkfgZYSAD0uYm7Vw4zm_tCswVQ"
```